src/gpu/vulkan/SDL_gpu_vulkan.c

```diff
@@ -668,6 +668,9 @@ typedef struct WindowData
     SDL_GPUSwapchainComposition swapchainComposition;
     SDL_GPUPresentMode presentMode;
     bool needsSwapchainRecreate;
+#ifdef SDL_PLATFORM_ANDROID
```

Comment:
Go ahead and remove `#ifdef SDL_PLATFORM_ANDROID` from around the uses of this variable, except for when it's set below.

Reply:
The memory block is initialized to 0, so this isn't needed.